app/assets/stylesheets/views/signin.scss

```diff
@@ -65,21 +65,91 @@
 // Registration View
 .registration {
   border-radius: unset;
-  padding: var(--su-6);
+  padding: var(--su-4);
 
   @media (min-width: $breakpoint-s) {
     border-radius: var(--radius);
     padding: var(--su-8);
+    padding-top: var(--su-6);
     margin: 0 auto;
     width: $breakpoint-s;
   }
 
+  &__button-container {
```

Comment:
@Ridhwana I have tried removing custom css as much possible but I was unable to use crayons-classes correctly mostly because I was unable to find any crayons which matches closely with these designs. The biggest outliers are font-size and colors. Therefore I have used utility-generated-css in html directly (similar to tailwind implementation).

Do let me know if it can be further improved.